python/src/aac/plugins/validators/_common.py

```diff
@@ -16,12 +19,30 @@ def get_validation_definition_from_plugin_definitions(plugin_definitions_string:
     Returns:
         The validation definition for the plugin.
     """
-    parsed_validator_definitions = parse(plugin_definitions_string)
-    validation_definitions = get_definitions_by_root_key("validation", parsed_validator_definitions)
+    return get_validation_definition_from_plugin_definitions(parse(plugin_definitions_string))
+
+
+def get_validation_definition_from_plugin_definitions(validator_definitions: list[Definition]) -> Definition:
+    """
+    Find the Definition associated with the specific validator plugin.
+
+    Args:
+        validator_definitions (list[Definition]): The parsed list of Definitions from the validator plugin.
+
+    Returns:
+        The validation definition for the plugin.
+    """
+    validation_definitions = get_definitions_by_root_key("validation", validator_definitions)
 
     if len(validation_definitions) != 1:
         raise PluginError(
             f"Expected one and only one validation definition.\nValidation Definitions:\n{validation_definitions}"
         )
 
     return validation_definitions[0]
+
+
+def get_plugin_validations_from_definitions(plugin_definitions: list[Definition], callback: Callable) -> list[ValidatorPlugin]:
+    """Return the ValidatorPlugins based on the plugin definitions."""
```

Comment:
I'm conflicted on this. The function name and return type both imply a list of validator plugins, but until we implement the ability to support multiple validators per plugin this function could easily lead users to incorrectly believe that they can implement multiple validators per plugin.

This note in the docstring should suffice for now, unless you'd prefer to alter the return type and use the following pattern in the plugins:
```python
plugin.register_validations([_get_plugin_validation()])
```


```suggestion
    """
    Return the ValidatorPlugins based on the plugin definitions.
    
    Beware, plugins currently only support one validator per plugin. 
    """
```